### DISCUSSION TITLE
Please add keyboard shortcut to change display

### DISCUSSION THREAD

**Discussion by Tomtim1** [rustdesk/rustdesk#4761] [Feature Request]
That would be a massive relief to be able to change display with CTRL+ALT+1 for screen one, CTRL+ALT+2 for second display etc.

**Comment by Gamedirection** [comment]
Although it may not fully meet your requirements, this serves as a temporary solution until the implementation is completed. 
[Discussion Comment - 7544763](https://github.com/rustdesk/rustdesk/discussions/1933#discussioncomment-7544763)

Additionally, it would be beneficial if users could seamlessly move the mouse across different screens and switch displays based on the monitor where the mouse is currently located. (This specifically would be nice for mobile) 

I would appreciate the addition of a "Next Screen" button that allows users to cycle through screens on mobile devices. This would streamline the process by reducing the number of clicks required and eliminate the need to check monitors remotely for programs that open. Thank you for considering this suggestion.
